include/cling/Interpreter/Interpreter.h

```diff
@@ -613,6 +613,7 @@ namespace cling {
 
     clang::CompilerInstance* getCI() const;
     clang::Sema& getSema() const;
+    IncrementalParser& getIncrParser() const { return *m_IncrParser; }
```

Comment:
Is this really necessary for internal code? The header still isn't public/published.
I've had other uses for getting the `IncrementalParser` and writing facades seems to double the complexity/dependencies rather than reduce it.